objects/tag.c

```diff
@@ -91,6 +91,83 @@
  *
  *    local t = awful.tag.find_by_name(awful.screen.focused(), "name")
  *
+ * **Common shortcuts**:
+ *
+ * Here is a few useful shortcuts not part of the default `rc.lua`. Add these
+ * functions above `-- {{{ Key bindings`:
+ *
+ * Delete the current tag
+ *
+ *    local function delete_tag()
+ *        local t = awful.screen.focused().selected_tag
+ *        if not t then return end
+ *        t:delete()
```

Comment:
Just me following my usual precondition programming style. In this case its a little silly since there is only one line, but these days I prefer to stick to that style. It avoids a lot of indentation, reduce NULL dereference risk and (but not in this case) make the code more readable.